app/src/pages/admin/serverSettingsPage/serverSettingsTabs/linksAndBrandingTab/addLinkForm/addLinkForm.jsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineMessages, useIntl } from 'react-intl';
+import { reduxForm } from 'redux-form';
+import { FieldProvider } from 'components/fields';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import classNames from 'classnames/bind';
+import { FieldText } from 'componentLibrary/fieldText';
```

Comment:
The "old-design" inputs should be used.
So, please, use input field from `components/inputs`.